index.html

```diff
@@ -4,20 +4,48 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="permaLogin.css">
+    <title>OpenBakalari login</title>
+    <style id="dyn"></style>
   </head>
-  <body>
     <script src="renderer.js"></script>
-    <header>
-      <h1>OpenBakaláři</h1>
-    </header>
+    <center id="errorstatus"></center>
     <main>
-      <form onsubmit="login()">
-        <input placeholder="Doména" type="url" name="domain" id="inst"><br>
-        <input placeholder="Uživatelské jméno" type="text" name="user" id="user"><br>
-        <input placeholder="Heslo" type="password" name="pass" id="pass"><br>
-        <input type="submit" value="Přihlásit se">
-      </form>
+      <header>
+        <center><h1>OpenBakaláři</h1></center>
+      </header>
+      <article>
+          <form onsubmit="login()">
+          <span class="hints">Domena: </span><br>
+          <input placeholder="Doména" type="url" name="domain" id="inst"><br>
+          <span class="hints">Uzivatelske jmeno: </span><br>
+          <input placeholder="Uživatelské jméno" type="text" name="user" id="user"><br>
+          <span class="hints">Heslo: </span><br>
+          <input placeholder="Heslo" type="password" name="pass" id="pass"><br>
+          <input type="submit" value="Přihlásit se">
+        </form>
```

Comment:
```suggestion
          <form onsubmit="login()">
          <input placeholder="Doména" type="url" name="domain" id="inst"><br>
          <input placeholder="Uživatelské jméno" type="text" name="user" id="user"><br>
          <input placeholder="Heslo" type="password" name="pass" id="pass"><br>
          <input type="submit" value="Přihlásit se">
        </form>
```
Už máme placeholder, to musí stačit.